routes/thoughtRoutes.js

```diff
@@ -0,0 +1,75 @@
+import express from "express";
+import listEndpoints from "express-list-endpoints";
+import { ThoughtModel } from "../models/ThoughtModel";
+
+const router = express.Router();
+
+// List of all endpoints
+router.get("/", async (req, res) => {
+  try {
+    res.status(200).send({
+      success: true,
+      message: "OK",
+      body: {
+        content: "Susannes Happy thoughts API",
+        endpoints: listEndpoints(router),
+      },
+    });
+  } catch (error) {
+    // Error handling
+    res.status(500).send({
+      success: false,
+      message: "Internal Server Error",
+      error: error.message,
+    });
+  }
+});
+
+// List of all thoughts - this should be limited to show only the 20 most recent thoughts showed in descending order
+router.get("/thoughts", async (req, res) => {
+  try {
+    const result = await ThoughtModel.find()
+      .sort({ createdAt: "desc" })
+      .limit(20);
+
+    res.status(200).json(result);
+  } catch (error) {
+    res.status(400).json({ message: "Bad request", error });
+  }
+});
+
+// Endpoint to post a new thought
+router.post("/thoughts", async (req, res) => {
+  try {
+    const newThought = new ThoughtModel(req.body);
+    newThought.hearts = 0; // Prevents the user from changing the heart
```

Comment:
This is a great security measure to adhere to the requirements, but I don't believe you actually need it since the default is already set to 0 in you model! 😃